pkgs/development/python-modules/canopen/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, nose
+, can
+, canmatrix }:
+
+buildPythonPackage rec {
+  pname = "canopen";
+  version = "0.5.1";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0806cykarpjb9ili3mf82hsd9gdydbks8532nxgz93qzg4zdbv2g";
+  };
+
+  # due to missing test/sample.eds
+  # https://github.com/christiansandberg/canopen/pull/57
+  doCheck = false;
```

Comment:
Thanks a lot for opening the PR! As long as no new version is published on PyPI, you could also rely on `fetchFromGitHub`.